src/runtime_src/core/common/api/xrt_xclbin.cpp

```diff
@@ -27,7 +27,10 @@
 #include "core/common/query_requests.h"
 #include "core/include/xclbin.h"
 #include "core/common/xclbin_parser.h"
+#include "xclbin_int.h"
```

Comment:
This should be second include.  xrt_xclbin.h and xclbin_int.h both are declarations of functions implemented in this file so should appear first.